examples/rich/rich.html

```diff
@@ -89,6 +95,20 @@
             }
           }
 
+          const customDraftBlockRenderMap = Immutable.Map({
+            'paragraph': Immutable.Map({
+              wrapper: null,
+              element: 'p'
+            }),
+            'unstyled': Immutable.Map({
+              wrapper: null,
+              element: 'p'
+            })
+          });
```

Comment:
I think this would be more appropriate as a `state` property, to indicate how it might be used (and imply how it may be updated) in practice. Also, then we don't have to create it in every render pass. :)